scripts/tests/chiptest/accessories.py

```diff
@@ -64,7 +64,7 @@ def killAll(self):
     def start(self, name, args):
         accessory = self.__accessories[name]
         if accessory:
-            # The args param comes directly from the sys.argv[1:] of Start.py and should contain a list of strings in
+            # The args param comes directly from the sys.argv[2:] of Start.py and should contain a list of strings in
```

Comment:
Any reason for this comment change? Was it stale before?

Reply:
Yes, the old comment was stale.